pkg/gitproviders/provider.go

```diff
@@ -118,7 +118,7 @@ func (p defaultGitProvider) CreateRepository(name string, owner string, private
 }
 
 func (p defaultGitProvider) DeployKeyExists(owner, repoName string) (bool, error) {
-	deployKeyName := "weave-gitops-deploy-key"
+	deployKeyName := "wego-deploy-key"
```

Comment:
Can we pull this into a constant to be shared between any function that needs it? If we want to change this in the future, someone will need to remember all the places were it is used.

As it is now, I call this the "magic string" pattern where by the app breaks if someone forgets to change the string in all the places.

Reply:
I know @jrryjcksn was doing something around the deploy key. @jrryjcksn are you changing any of this? If not i can just move it to a const.